tests/Feature/WebRouteTest.php

```diff
@@ -81,6 +83,36 @@ public function test_results_route()
             });
     }
 
+    /**
+     * Test that the /results response contains mismatch data
+     *
+     *  @return void
+     */
+    public function test_results_route_retrieves_mismatches()
+    {
+        $import = ImportMeta::factory()
+        ->for(User::factory()->uploader())
+        ->create();
+
+        $mismatch = Mismatch::factory()
+            ->for($import)
+            ->create();
+
+        $response = $this->get(route('results', [
+            'ids' => $mismatch->item_id
+        ]));
+
+        $response->assertSuccessful();
+        $response->assertViewIs('app')
+            ->assertInertia(function (Assert $page) use ($mismatch, $import) {
+                $page->component('Results')
+                    ->has('results.0', 11)
```

Comment:
Hello, where is that 11 coming from???

Reply:
that's just the size of the results object, I counted the number of properties

Reply:
Sure, feel free - it's just a suggestion. From my experience it makes sense to pinpoint the object size in such a test, because once something changes, the test will fail and indicate that maybe other modifications to tests are needed, too. Also the error message is straightforward: `Inertia property [results.0] does not have the expected size.`

Reply:
Could you add the comment? Then I can approve. Thanks :)

Reply:
nevermind. rebase is not needed